components/announcement_bar/purchase_now_link/purchase_now_link.tsx

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {trackEvent} from 'actions/telemetry_actions';
+
+export interface Props {
+    buttonTextElement: JSX.Element;
+}
+
+const PurchaseNowLink: React.FC<Props> = (props: Props) => {
```

Comment:
Would prefer to simplify name.
```suggestion
const PurchaseLink: React.FC<Props> = (props: Props) => {
```